Building.md

```diff
@@ -0,0 +1,49 @@
+This is the building guide for FullScreenShenanigans projects.
+
+### Table of Contents
+
+1. [Obtaining the Source Code](#obtaining-the-source-code)
+2. [Build Process](#build-process)
+3. [Using the Console](#using-the-console)
+
+## Obtaining the Source Code
+
+In order to play a FullScreenShenanigans game, you first need to obtain the source code. There are two ways to do this:
+
+A. Clone the project repository using GitHub:
```

Comment:
A. and B. aren't supported by Markdown (or at least not on GitHub). Turn this into #### headers instead.